CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ All notable changes to this project will be documented in this file.
     * #### Added
     * #### Changed
         * Fix [#1491](https://github.com/ni/nimi-python/issues/1491): import_attribute_configuration_buffer() fails intermittently when `list` or `array.array` is passed in.
+        * Update "Driver Version Tested Against", in documentation, with latest versions installed on nimi-bot. The version is 20.5.0 for NI-DCPower, NI-SWITCH, and NI-DMM. no changes on other drivers
```

Comment:
Do we always call this out? It seems to be irrelevant to the CHANGELOG.md.
If we do always call it out, then keep it.

Reply:
I suppose it's fine, it tells the client that we have already tested it against these versions. What do you think Shreyas?

Reply:
It seems unnecessary, but you could also argue it's technically correct since we are changing user-facing documentation. Also, we have done it many times in the past. I'd say we keep it, don't think anyone will complain about us providing this extra info.